Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -200,15 +200,15 @@ void GeneralWidget::AddDescriptions()
 #if defined(_WIN32)
```

Comment:
Why do we even have two different strings for Windows and non-Windows? Can't we just unify both with "any of the other backends"?

The non-Windows description is also missing the "Different games and different GPUs will behave differently" bit.

Reply:
Done.

I originally did add the "Different games" bit to non-Windows platforms, but apparently that got lost while I was rewriting the history before the PR.